tools/python/testserver.py

```diff
@@ -171,7 +171,7 @@ def dispatch_response(self, payload):
             self.send_header(h, payload.headers()[h])
         self.send_header("Content-Length", payload.length())
         self.end_headers()
-        self.wfile.write(payload.message())
+        self.wfile.write(payload.message().encode('utf8'))
```

Comment:
What is the default encoding? Why explicit encode is needed?

Reply:
`payload.message()` method returns string, while `write` method expects bytes object. Call to `encode` is needed to convert string -> bytes.